src/sentry/sentry_metrics/indexer/ratelimiters.py

```diff
@@ -19,15 +19,11 @@
 
 
 def _build_quota_key(use_case_id: UseCaseKey, org_id: Optional[OrgId] = None) -> str:
-    use_case_str = {
-        UseCaseKey.PERFORMANCE: "performance",
-        UseCaseKey.RELEASE_HEALTH: "releasehealth",
-    }[use_case_id]
```

Comment:
Don't we need to keep these names as before? Otherwise the quotas we set would be lost.

Reply:
oh yeah sorry forgot to add em back in